gulpfile.js

```diff
@@ -130,6 +131,30 @@ const { launch, connect } = require('hadouken-js-adapter');
 	// This will get set when the container (Electron or Openfin) is launched. This is used to calculate how long it takes to start up the app.
 	let launchTimestamp = 0;
 
+	/**
+	 * Mody 10/04/2019
+	 * Reads installed Electron's version from FEA repo.
+	 * Another option is to export electron's version in
+	 * deploymentHelpers in FEA. However I'm just avoiding 2 PRs
+	 */
+	const getElectronVersion = () => {
+		// You may run `npm run dev` before running `npm i` inside 
+		// finsemble-electron-adapter in that case, the electron
+		// module does not exists.
+		try {
+			const packageFile = require(
+				path.join(
+					FEA_PATH,
+					'node_modules',
+					'electron',
+					'package.json')
+			)
+			return packageFile.version
```

Comment:
normally we use semi colons (also on line 155)

Reply:
I'm sorry, trying to get used to them, I will add them.

Reply:
Added semicolons.